### DISCUSSION TITLE
Time based password

### DISCUSSION THREAD

**Discussion by depperm** [Divide-By-0/ideas-for-projects-people-would-use#8] [Show and tell]
I created an example of a time based password [here](https://depperm.github.io) (third tab). I listed problems/risks I see with such a solution and also benefits.

Let me know about any improvements, questions or bugs.

**Comment by Divide-By-0** [comment]
Hmm not sure this is what I meant; I was trying to suggest a hashless simple arithmetic function that would be robust to looking over shoulder -- all I can see is a hashed function based on the current time/date numbers; perhaps I'm looking at the wrong thing? (edited)

**Reply by depperm** [reply]
For instance, if the current time is AB:CD, the user can set their password as say, (A+B), (|C-D|), (A * B % 10), (D)

**Reply by Divide-By-0** [reply]
Oh! Sorry I totally missed the third tab on mobile somehow -- is there some android/iphone app I can get?

**Reply by depperm** [reply]
Currently there isn't an app, this is just a proof of concept. Is this what you meant (password wise, not device implementation)?

**Reply by Divide-By-0** [reply]
Looks great! Exponents and modulo functions would be interesting (or honestly having another option to let the user write their own python function inputting a time/date array and outputting the password array would be sick haha).

I know there are a number of time based password lock apps already; modifying an already open sourced one might be the fastest way to do this! I probably will only give out the bounty for a fully working app :)
